lab_1_classify_by_unigrams/main.py

```diff
@@ -12,6 +12,16 @@ def tokenize(text: str) -> list[str] | None:
     :return: a list of lower-cased tokens without punctuation
     """
 
+    if isinstance(text, str):
+        text_lower = text.lower()
+    
+        for symbol in text_lower:
+            if not symbol.isalpha():
+                text_lower = text_lower.replace(symbol, '')
+    
+        tokens = list(text_lower)
+    else:
+        return None
```

Comment:
Consider doing an early return: when incorrect value is met, escape the function immediately, this helps reduce the indentation